src/test/java/org/mule/module/xml/validator/SchemaValidationTestCase.java

```diff
@@ -37,11 +41,15 @@
   public SystemProperty externalEntities = new SystemProperty(EXTERNAL_ENTITIES_PROPERTY, "true");
 
   private static final String SIMPLE_SCHEMA = "validation/schema1.xsd";
+  private static final String ALTERNATIVE_SIMPLE_SCHEMA = "validation/schema2.xsd";
   private static final String INCLUDE_SCHEMA = "validation/schema-with-include.xsd";
 
   private static final String VALID_XML_FILE = "validation/validation1.xml";
+  private static final String VALID_ALTERNATIVE_XML_FILE = "validation/validation4.xml";
   private static final String INVALID_XML_FILE = "validation/validation2.xml";
 
+  private static final int NUM_CONCURRENT_VALIDATIONS = 1000;
```

Comment:
isn't this overdoing it? isn't 10 or 50 enough?

Reply:
I'll move this to munit.